server/services/reportService.test.ts

```diff
@@ -0,0 +1,29 @@
+import { Response } from 'express'
+import { createMock } from '@golevelup/ts-jest'
+
+import ReportClient from '../data/reportClient'
+import ReportService from './reportService'
+
+describe('ReportService', () => {
+  const reportClient = new ReportClient(null) as jest.Mocked<ReportClient>
+  const reportClientFactory = jest.fn()
+
+  const service = new ReportService(reportClientFactory)
+
+  const token = 'SOME_TOKEN'
+
+  beforeEach(() => {
+    jest.resetAllMocks()
+    reportClientFactory.mockReturnValue(reportClient)
+  })
+
+  describe('getReport', () => {
+    it('calls the getApplicationsReport client method when the reportType is "applications" pipes the report to the response', () => {
```

Comment:
is this test description accurate? Do we need to remove `when the reportType is "applications"`?

Reply:
thks, that's some copy/paste cruft, will tidy